frontend/explorer/public/hash.html

```diff
@@ -26,6 +26,19 @@
 		<h1 class='banner-head' id='page-title'>ThreeFold Chain Hash Explorer</h1>
 	</div>
 
+	<div id='page-navigation'>
+		<p id="nav-links">
+			<span class="grey-text">
+				You are here:
+				<a href='index.html'>Home</a>
+			</span>
+			>
+		</p>
+	</div>
+
+	<div id='page-navigation'>
```

Comment:
You forgot to remove the original div that you added for this feature. Now you have 2 divs with the same ID.

Reply:
Fixed